### PR TITLE
lbrynet: init at 0.9.1

### PR DESCRIPTION
**Pull request by benley**
###### Motivation for this change
https://lbry.io/news/20000-illegal-college-lectures-rescued

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -173,6 +173,15 @@ in {
     };
   };
 
+  base58 = buildPythonPackage rec {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: can you store new python packages in `./pkgs/development/python-modules` and only reference them here? This is the new policy as the file has grown too big.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -173,6 +173,15 @@ in {
...
+    name = "base58-0.2.4";
+
+    srcs = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Does `fetchPypi` works here?

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -230,6 +239,16 @@ in {
     };
   };
 
+  envparse = buildPythonPackage rec {
+    name = "envparse-0.2.0";
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Does `fetchPypi` works here?

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -7993,6 +8059,15 @@ in {
     };
   };
 
+  jsonrpc = buildPythonPackage rec {
+    name = "jsonrpc-1.2";
+
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: see above

File: pkgs/top-level/python-packages.nix [hunk 5]
```diff
@@ -8349,6 +8424,15 @@ in {
     };
   };
 
+  miniupnpc = buildPythonPackage rec {
+    name = "miniupnpc-1.9";
+
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: see above

File: pkgs/top-level/python-packages.nix [hunk 6]
```diff
@@ -9937,6 +10021,24 @@ in {
     };
   };
 
+  seccure = buildPythonPackage rec {
+    name = "seccure-0.3.2";
+
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: see above.

File: pkgs/top-level/python-packages.nix [hunk 7]
```diff
@@ -22265,6 +22367,18 @@ in {
 
   requests_oauthlib = callPackage ../development/python-modules/requests-oauthlib.nix { };
 
+  requests-futures = buildPythonPackage rec {
+    name = "requests-futures-0.9.7";
+
+    propagatedBuildInputs = with self; [ requests futures ];
+
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: see above

**Review by Mic92 — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (benley) [inline pkgs/top-level/python-packages.nix]: Oh yes definitely, I would prefer it that way.  I thought things were moving in the opposite direction!

**Comment by FRidh** [comment]
This has been inactive for some time so I am closing this.

**Pull request closed without merging**